File: zt-hook/src/main/java/com/zzt/zt_hook/MainActivity.java
package com.zzt.zt_hook;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.zzt.appframe9.BuildConfig;

public class MainActivity extends AppCompatActivity {

    BuildConfig cc ;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: app/src/main/java/com/zzt/appframe9/MainActivity.kt
package com.zzt.appframe9

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.WindowManager

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        getWindow().setFlags(
            WindowManager.LayoutParams.FLAG_SECURE,
            WindowManager.LayoutParams.FLAG_SECURE
        );
        setContentView(R.layout.activity_main)
    }
}